react/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
remove

Reply:
Should be resolved in https://github.com/radiasoft/sirepo/pull/4667/commits/9cc4554f84321a4ebad33583e8e797f7fec20a7d